torch/onnx/__init__.py

```diff
@@ -138,6 +137,36 @@
 producer_name = "pytorch"
 producer_version = _C_onnx.PRODUCER_VERSION
 
+_CUSTOM_ONNX_DECOMPS = {}
+
+
+def register_decomp(
+    target: Callable, onnx_decomp: Callable, is_complex: bool = False
+) -> None:
+    """Register a function to decompose a PyTorch node to ONNX.
+
+    Args:
+        target: The PyTorch node callable target. E.g. ``torch.ops.aten.add``.
+        onnx_decomp: The ONNX decomposition function to register.
```

Comment:
Is the notion of ONNX decomposition function documented/explained anywhere? Especially, given the older symbolic functions and the new onnxscript ones.